### DISCUSSION TITLE
is it possible to set bootstrapServers from a secret?

### DISCUSSION THREAD

**Discussion by alejandrade** [strimzi/strimzi-kafka-operator#5213] [Q&A]
Just a general question. 

I want to set bootstrapServers from a secret.

  bootstrapServers: "${directory:/opt/kafka/external-configuration/secrets:kafka_bootstrap_brokers}"
 did not seem to work

**Comment by scholzj** [comment]
No, that is not possible. The bootstrap server is always specified as part of the custom resource.
